static/js/containers/SearchPage_test.js

```diff
@@ -271,7 +279,8 @@ describe("SearchPage", () => {
     assert.deepEqual(inner.state(), {
       // Because this is non-incremental the previous from value of 7 is replaced with 0
       from: 0,
-      text: undefined
+      text: undefined,
+      upvotedPosts
```

Comment:
if it's not too much trouble it might be good to add a test which just makes sure that the post we expect is added to the `upvotedPosts` map